models/balancer/base/balancer_v2_base_bpt_prices.sql

```diff
@@ -1,161 +1,16 @@
+{% set blockchain = 'base' %}
+
 {{
     config(
-        schema = 'balancer_v2_base',
+        schema='balancer_v2_' + blockchain,
```

Comment:
I think we may need to hardcode the schema here.

Reply:
If this solved the problem, you can change schema in other sqls as well.

Reply:
Also force-push may override the fixing commits I tried to submit.
But I think the main issue would be the schema.